components/search/TierToggler.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div>
+    <p class="font-italic">
```

Comment:
Let's add a "mb-5" class here, to make the paragraph more distinct from the pagination and having it more in line with the design.